### ISSUE TITLE
Keyboard Navigation does not work for fourth space

### ISSUE THREAD

**Issue by HarHarLinks** [vector-im/element-web#19255]
### Steps to reproduce

![image](https://user-images.githubusercontent.com/2803622/135711044-aa204ce5-59c8-43b7-b5f5-039d2741c465.png)

### What did you expect?/What happened?

I can navigate to Home, :house:, and F space using Ctrl+1/2/3 respectively no problem, but Ctrl+4 for matrix community space does nothing.

### Related
#17776

### Operating system

arch

### Application version

Element Nightly version: 2021100201 Olm version: 3.2.3

### How did you install the app?

aur/nightly-bin

### Homeserver

private

### Have you submitted a rageshake?

No

**Comment by t3chguy** [comment]
I have a similar issue with Cmd+6 on Macos

**Comment by t3chguy** [comment]
ah, the general case is you can't access your last space because of an off-by-one, will fix
